test/containers/trends/TrendsSVGContainer.test.js

```diff
@@ -330,5 +391,24 @@ describe('TrendsSVGContainer', () => {
         expect(noDataWrapper.find(NoData).prop('dataType')).to.equal('smbg');
       });
     });
+
+    describe('interactions', () => {
+      describe('clicking on the SVG', () => {
+        it('should fire the click handler', () => {
+          // we have to stub here instead of spying because we're not doing a full mount
+          // and so the click handler's inspection of the event target will fail w/a spy
+          const clickStub = sinon.stub(TrendsSVGContainer.prototype, 'handleClick');
+          const stuckDatesProps = _.assign(
+            {}, props, { stuckCbgDateTraces: ['2017-01-01'] },
+          );
+          const stuckDatesWrapper = shallow(<TrendsSVGContainer {...stuckDatesProps} />);
+          expect(clickStub.callCount).to.equal(0);
+          const svg = stuckDatesWrapper.find('svg');
+          svg.simulate('click');
```

Comment:
We can still use a `sinon.spy` here if we pass in an object for the click event. like:
`svg.simulate('click', { target: { id: 'cbgSlice' } });`
see: https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/simulate.md for details.